eslint-plugin-prettier.js

```
@@ -140,6 +140,18 @@ const eslintPluginPrettier = {
         'prefer-arrow-callback': 'off',
       },
     },
+    'flat/recommended': {
+      plugins: {
+        get prettier() {
+          return eslintPluginPrettier
```

ESLint

[error] Insert `;`

(prettier/prettier)